src/applications/personalization/dashboard/components/benefit-payments-and-debts/BenefitPaymentsAndDebt.jsx

```diff
@@ -68,13 +71,16 @@ const BenefitPaymentsAndDebt = ({
                 icon="user-check"
                 newTab
                 text="View your payment history"
+                /* eslint-disable react/jsx-no-bind */

```

Comment:
onClick param does require a callback-function, so I'm disabling the eslint-warning.